exporting/prometheus/remote_write/remote_write.c

```diff
@@ -115,22 +99,36 @@ int init_prometheus_remote_write_instance(struct instance *instance)
     instance->end_host_formatting = NULL;
     instance->end_batch_formatting = format_batch_prometheus_remote_write;
 
-    instance->send_header = prometheus_remote_write_send_header;
+    instance->prepare_header = prometheus_remote_write_prepare_header;
     instance->check_response = process_prometheus_remote_write_response;
 
     instance->buffer = (void *)buffer_create(0);
     if (!instance->buffer) {
```

Comment:
Please, remove this unnecessary test as described before.

Reply:
Removed.